packages/superset-ui-encodeable/src/parsers/createGetterFromChannelDef.ts

```diff
@@ -0,0 +1,14 @@
+import { get } from 'lodash/fp';
+import identity from '../utils/identity';
+import { ChannelDef } from '../types/ChannelDef';
+import { isValueDef } from '../typeGuards/ChannelDef';
+
+export default function createGetterFromChannelDef(definition: ChannelDef): (x?: any) => any {
```

Comment:
Could this function have more explicit typing? If not, would be good to leave a comment documenting expected input/output